daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1104,36 +1099,11 @@ void IEEE1588Port::processEvent(Event e)
 			   system_time, local_system_freq_offset, sync_count,
 			   pdelay_count, port_state, asCapable );
 
-			if (!automotive_profile) {
-				/* If accelerated_sync is non-zero then start 16 ms sync
-				   timer, subtract 1, for last one start PDelay also */
-				if( _accelerated_sync_count > 0 ) {
-					clock->addEventTimerLocked
-					  ( this, SYNC_INTERVAL_TIMEOUT_EXPIRES, 8000000 );
-					    --_accelerated_sync_count;
-				} else {
-					syncDone();
-					if( _accelerated_sync_count == 0 ) {
-						--_accelerated_sync_count;
-					}
-					wait_time *= 1000; // to ns
-					wait_time =
-					  ((long long)
-					     (pow((double)2,getSyncInterval())*1000000000.0)) -
-					    wait_time;
-					wait_time = wait_time > EVENT_TIMER_GRANULARITY ? wait_time :
-					            EVENT_TIMER_GRANULARITY;
-					startSyncIntervalTimer(wait_time);
-				}
-			}
-			else {
-				// Automotive Profile
-				syncDone();
+			syncDone();
 
```

Comment:
I did see the wait time adjustment math. I concluded it was most relevant to the automotive fast sync case where TX timestamp read time might have a large effect percentage wise on the Sync interval timing. I didn't think removing the adjustment would have any negative operational effect - although I must admit I haven't hooked up test equipment to try to figure out if there is an such effect.